utils/experiment-examples/experiment-1.json

```diff
@@ -0,0 +1,54 @@
+{
+    "experiment": {
+        "rounds": [
```

Comment:
Add `email` and `description`. In the description, provide an example of what will happen. Overall, make the example realistic (this one seems to make the exact same announcement 4 times, for no apparent reason).

One possible example would be to use two prefixes to test propagation delay of an announcement from `ufmg01`, `ams01`, `seattle01` and anycast. This would require 3 rounds and 2 prefixes.